javascript.js

```diff
@@ -0,0 +1,20 @@
+// set variavles
+var user = []; //inline coment
```

Comment:
Typo inside comment in token 'coment'. Possible corrections:comment (37),moment (30),cement (26)